api/libproject/environments.rb

```diff
@@ -26,6 +26,7 @@ def env_with_defaults(env, config)
     :cdr_sql_instance => "workbench",
     :source_cdr_project => "all-of-us-ehr-dev",
     :source_cdr_wgs_project => "all-of-us-workbench-test",
+    :tanagra_tag => "0.0.219",
```

Comment:
Q: How do I deploy my local-branch of tanagra during development?
Does -v accept branch-name?

Reply:
Currently it only accepts a tag, but I can add an optional arg to take a branch name